ecosystem/typescript/sdk/examples/javascript/index.js

```diff
@@ -39,6 +39,13 @@ const aptosCoin = "0x1::coin::CoinStore<0x1::aptos_coin::AptosCoin>";
   accountResource = resources.find((r) => r.type === aptosCoin);
   console.log(`account2 coins: ${accountResource.data.coin.value}. Should be 717!`);
 
+  const provider = new aptos.Provider(aptos.Network.DEVNET);
+  console.log("\n=== Checking if indexer devnet chainId same as fullnode chainId  ===");
+  const indexerLedgerInfo = await provider.getIndexerLedgerInfo();
+  const fullNodeChainId = await provider.getChainId();
+
+  if (indexerLedgerInfo.ledger_infos[0].chain_id !== fullNodeChainId) return;
```

Comment:
Let's please also log this, because it's basically skipping the rest of the test right?

Reply:
Done